src/platforms/apple/common/usage.mdx

```diff
@@ -53,4 +53,5 @@ This integration will swizzle some methods to create breadcrumbs e.g.: for `UIAp
 
 ## Known Limitations
 
+- If you're currently using a version between 6.0.2 and 6.0.8 we strongly recommend you to upgrade it to the latest. In those versions the [SDK had a regression](https://github.com/getsentry/sentry-cocoa/pull/786) that affected the quality of the stacktraces and error messages. We fixed this in 6.0.9.
```

Comment:
```suggestion
- If you're currently using a version between 6.0.2 and 6.0.8, we **highly recommend** you upgrade to the latest version to avoid [a regression](https://github.com/getsentry/sentry-cocoa/pull/786) that affected the quality of the stacktraces and error messages, which was fixed in 6.0.9.
```